packages/suite/src/components/guide/GuideHeaderBreadcrumb.tsx

```diff
@@ -0,0 +1,103 @@
+import * as guideActions from '@suite-actions/guideActions';
+import { variables } from '@trezor/components';
+import { Translation } from '@suite-components';
+import { findParentCategories, getItemTitle } from '@suite-utils/guide';
+
+const BreadcrumbWrapper = styled.span`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const PreviousCategoryLink = styled.a`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+
+    &:hover {
+        color: ${props => darken(props.theme.HOVER_DARKEN_FILTER, props.theme.TYPE_LIGHT_GREY)};
+    }
+`;
+
+const BreadcrumbDelimiter = styled.span`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    margin: 0 5px;
+`;
+
+const CategoryLink = styled.a`
+    color: ${props => props.theme.TYPE_GREEN};
+
+    &:hover {
+        color: ${props => darken(props.theme.HOVER_DARKEN_FILTER, props.theme.TYPE_GREEN)};
+    }
+`;
+
+const GuideHeaderBreadcrumb = () => {
+    const { language, item, root } = useSelector(state => ({
+        item: state.guide.item,
+        language: state.suite.settings.language,
+        root: state.guide.root,
+    }));
+
+    const { setView, openItem } = useActions({
+        setView: guideActions.setView,
+        openItem: guideActions.openItem,
+    });
+
+    const fallbackBreadcrumb = (
+        <BreadcrumbWrapper>
+            <CategoryLink onClick={() => setView('GUIDE_DEFAULT')}>
+                <Translation id="TR_GUIDE_DASHBOARD" />
+            </CategoryLink>
+        </BreadcrumbWrapper>
+    );
+
+    if (!item || !root) return fallbackBreadcrumb;
+
+    const parentCategories = findParentCategories(item, root);
+
+    if (!parentCategories?.length) return fallbackBreadcrumb;
+
+    const parentCategory = parentCategories[parentCategories.length - 1];
+    const parentCategoryTitle = getItemTitle(parentCategory, language);
+
+    let grandParentCategory: GuideCategoryType;
+    let grandParentCategoryTitle;
+    if (parentCategories.length > 1) {
+        grandParentCategory = parentCategories[parentCategories.length - 2];
+        grandParentCategoryTitle = getItemTitle(grandParentCategory, language);
+    }
+
+    return (
+        <BreadcrumbWrapper>
+            <PreviousCategoryLink
+                onClick={() => {
+                    if (grandParentCategory) {
+                        openItem(grandParentCategory);
+                    } else {
+                        setView('GUIDE_DEFAULT');
+                    }
+                }}
+            >
+                {grandParentCategoryTitle || <Translation id="TR_GUIDE_DASHBOARD" />}
+            </PreviousCategoryLink>
+            <BreadcrumbDelimiter>/</BreadcrumbDelimiter>
+            <CategoryLink
+                onClick={() => {
+                    if (grandParentCategory) {
+                        openItem(grandParentCategory);
+                    } else {
+                        openItem(parentCategory);
```

Comment:
It will be `grandParentCategoryTitle` / `parentCategoryTitle` but pointing to same view because of this type of layout.
![Screenshot 2021-06-30 at 14 38 20](https://user-images.githubusercontent.com/33235762/123961613-d53dca00-d9b0-11eb-9940-95583c6b7ab2.png)

Reply:
I'm getting lost in what should be rendered how and where. I guess I'll just believe in you. .)